R/correctionsataglance-data.R

```diff
@@ -342,13 +344,15 @@ findTextLocations <- function(dataIn, isDateData = FALSE, ...){
   return(list(x = x, y = y))
 }
 
-isTextLong <- function(labelText, dateLim, startD, endD){
-  early <- which(startD < dateLim[1])
-  late <- which(endD > dateLim[2])
-  startD[early] <- dateLim[1]
-  endD[late] <- dateLim[2]
+isTextLong <- function(labelText, dateLim = NULL, startD, endD, totalDays = NULL){
+  if(is.null(totalDays)){
+    early <- which(startD < dateLim[1])
+    late <- which(endD > dateLim[2])
+    startD[early] <- dateLim[1]
+    endD[late] <- dateLim[2]
+    totalDays <- difftime(dateLim[2], dateLim[1], units="days")
+  } 
   
-  totalDays <- difftime(dateLim[2], dateLim[1], units="days")
   widthOfChar <- (1/365)*totalDays*3 #each character will be 1/365 * num days in the range * buffer
   widthOfLabel <- nchar(labelText)*widthOfChar
   widthOfRect <- difftime(endD, startD, units="days")
```

Comment:
is this out on dev? Just ran a report that has lots of months but they are all labeled, and its still crazy. I think this might be a diff ticket

https://cida-eros-aqcudev.er.usgs.gov:8443/aqcu-webservice/service/reports/correctionsataglance/?primaryTimeseriesIdentifier=1343310484b0434c83acacbe56fca350&station=01047200&startDate=2010-10-01&endDate=2013-09-30